Introductory/projects/EngineMonitorProject.json

```diff
@@ -2,68 +2,83 @@
+    "target" : "procedure/com.vantiq.engines.EngineMonitor.speedUpdate"
+  }, {
+    "source" : "source/com.vantiq.engines.SpeedSensor",
+    "target" : "app/com.vantiq.engines.IngestExternalSensorEventsService.IngestExternalEvents0"
+  }, {
+    "source" : "source/com.vantiq.engines.TemperatureSensor",
+    "target" : "app/com.vantiq.engines.IngestExternalSensorEventsService.IngestExternalEvents1"
   }, {
     "source" : "services/com.vantiq.engines.EngineMonitor",
-    "target" : "topic//com.vantiq.engines.EngineMonitor/TempEvent/handler/inbound"
+    "target" : "type/com.vantiq.engines.SpeedEvent"
+  }, {
+    "source" : "client/com.vantiq.engines.EngineMonitor",
+    "target" : "services/com.vantiq.engines.EngineMonitor"
   }, {
-    "source" : "app/com.vantiq.engines.EngineMonitor_SpeedEvent_EventHandler",
-    "target" : "topic//outbound/engine/speed"
+    "source" : "app/com.vantiq.engines.IngestExternalSensorEventsService.IngestExternalEvents1",
+    "target" : "source/com.vantiq.engines.TemperatureSensor"
   }, {
-    "source" : "app/com.vantiq.engines.EngineMonitor_TempEvent_EventHandler",
-    "target" : "topic//outbound/engine/temp"
+    "source" : "app/com.vantiq.engines.IngestExternalSensorEventsService.IngestExternalEvents0",
+    "target" : "source/com.vantiq.engines.SpeedSensor"
   }, {
-    "source" : "client/EngineSimulation",
-    "target" : "topic//my/response/topic"
+    "source" : "designmodel/com.vantiq.engines.EngineMonitor",
+    "target" : "client/com.vantiq.engines.EngineMonitor"
+  }, {
+    "source" : "designmodel/com.vantiq.engines.EngineMonitor",
+    "target" : "services/com.vantiq.engines.EngineMonitor"
   } ],
   "name" : "EngineMonitorProject",
   "options" : {
-    "description" : "The Engine Monitor tutorial walks through developing and testing a Vantiq Service to monitor the speed and temperature of an engine. \nThe SpeedEvent and TempEvent Inbound Event Types are implemented with Visual Event Handlers to process the stream of sensor readings and detect malfunction conditions in the engine. Outbound EngineMalfunction Events are produced if a malfunction is detected. A Client is used\nto visualize the data and display any alerts as they occur.\nUse the Event Generator to simulate sensor events.",
+    "description" : "The Engine Monitor tutorial walks through developing and testing a Vantiq Design Model to monitor the speed and temperature of an engine. The SpeedEvent and TemperatureEvent Inbound Event Types are implemented with Visual Event Handlers to process the stream of sensor readings and detect malfunction conditions in the engine. A Client is used to visualize the data and display any alerts as they occur.\n\n**Please note**: the engine sensor data is read from two public MQTT Sources that are disabled by default in this Project. Use the Active Resource Control Center (seen below) to activate the *SpeedSensor* and *TemperatureSensor* Sources by clicking their **Active** slider.",
```

Comment:
I see that you can have the Active Resource control pane open in the imported project layout.  Nice!